lib/hyrax/hirmeos/client.rb

```diff
@@ -19,7 +19,7 @@ def get_work(uuid)
   end
 
   def request_token
-    response = Faraday.post(URI.join(token_base_url, 'tokens')), { email: username, password: password }.to_json
+    response = Faraday.post(URI.join(token_base_url, 'tokens'), { email: username, password: password }.to_json)
     JSON.parse(response[0].body)['data'][0]['token']
```

Comment:
I think this line needs to change now:
```suggestion
    JSON.parse(response.body)['data'][0]['token']
```